rosbag2_py/src/rosbag2_py/_message_definitions.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright 2024 Intrinsic Innovcation LLC. All rights reserved.
```

Comment:
Typo. Should be "Innovation"
```suggestion
// Copyright 2024 Intrinsic Innovation LLC. All rights reserved.
```

Reply:
Oops, thanks!